torchgeo/datasets/vhr10.py

```diff
@@ -297,6 +291,7 @@ def _load_image(self, id_: int) -> Tensor:
             tensor = torch.from_numpy(array)
             # Convert from HxWxC to CxHxW
             tensor = tensor.permute((2, 0, 1))
+            tensor = tensor.float() / tensor.max()
```

Comment:
I don't think we want to do normalization like this, it should be left up to the user.